reactive/etcd.py

```diff
@@ -518,9 +518,20 @@ def render_default_user_ssl_exports():
     client_crt = opts['client_certificate_path']
     client_key = opts['client_key_path']
 
-    evars = ['export ETCDCTL_KEY_FILE={}\n'.format(client_key),
-             'export ETCDCTL_CERT_FILE={}\n'.format(client_crt),
-             'export ETCDCTL_CA_FILE={}\n'.format(ca_path)]
+    major, minor, _ = etcd_version().split('.')
+
+    if int(major) >= 3 and int(minor) >= 3:
+        evars = [
+            'export ETCDCTL_KEY={}\n'.format(client_key),
+            'export ETCDCTL_CERT={}\n'.format(client_crt),
+            'export ETCDCTL_CACERT={}\n'.format(ca_path)
+        ]
+    else:
+        evars = [
+            'export ETCDCTL_KEY_FILE={}\n'.format(client_key),
+            'export ETCDCTL_CERT_FILE={}\n'.format(client_crt),
+            'export ETCDCTL_CA_FILE={}\n'.format(ca_path)
+        ]
```

Comment:
This code will need to be re-run if the etcd version changes, i.e. when `config.changed.channel`